admin/client/App/screens/List/actions/active.js

```diff
@@ -73,8 +73,18 @@ export function clearAllFilters () {
 	};
 }
 
+// This is being used on first page load to set all filters from params
+export function setActiveFilters (filters) {
```

Comment:
It's doing the same work as other reads from params, which are all being done in sagas, so I was following the established pattern.